packaging/generic-rpm/amazon-ecs-init.spec

```diff
@@ -0,0 +1,99 @@
+# Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: The build tag is specified to al2 because if unspecified it defaults to what al1 expects, but
+# the al2 config is the one that specifies the correct cgroup for all the rpm based system. Should probably
+# refactor the code to fix this logic.
+%global gobuild_tag al2
```

Comment:
Can we change this now since Cam made the change for ubuntu tag? Or it is still not feasible?

Reply:
no build tag defaults to al1 logic, so we still need to avoid it. i added a separate config file for generic rpm https://github.com/aws/amazon-ecs-init/pull/389/files#diff-27b431da0991cf90dec63480aafc5e13631b908f3ed44df8c7e9485dc16c6b24R1, to avoid abusing the al2 tag